lib/index.js

```diff
@@ -82,18 +86,24 @@ Optimizely.prototype.initialize = function() {
  */
 
 Optimizely.prototype.track = function(track) {
-  var payload = {};
+  // Use the new-style API (which is compatible with Classic and X)
+  var payload = {
+    type: 'event',
+    eventName: track.event(),
+    tags: track.properties()
+  };
   var revenue = track.revenue();
```

Comment:
yes, so you want to remove `tags.revenue` if it exists. Normally, we just set the revenue by pulling `track.revenue()` to whatever the semantic revenue property for the tool is and then we would `var props = track.properties();` and `remove(props, 'revenue');`